hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```
@@ -443,7 +428,7 @@ private synchronized boolean getNamespaceEnabledInformationFromServer(
    */
   @VisibleForTesting
   boolean isNamespaceEnabled() throws TrileanConversionException {
-    return this.isNamespaceEnabled.toBoolean();
+    return abfsConfiguration.getIsNamespaceEnabledAccount().toBoolean();
```

here also should we have a try-catch block for TrileanConversionException like we have in AbfsClient-line 1722?

---

This method should throw an exception if namespace enabled isn't initialized, so a try-catch block isn't necessary here.